.github/workflows/package-regression.yml

```
@@ -83,3 +83,75 @@ jobs:
       DC_GITHUB_CLIENT_ID: ${{ secrets.DC_GITHUB_CLIENT_ID }}
       DC_GITHUB_CLIENT_SECRET: ${{ secrets.DC_GITHUB_CLIENT_SECRET }}
       DC_GITHUB_REDIRECT_URI: ${{ secrets.DC_GITHUB_REDIRECT_URI }}
+
+  scheduled-status:
+    name: scheduled-regression-status
+    if: ${{ always() && github.event_name == 'schedule' }}
+    concurrency:
+      group: scheduled-package-regression-issue
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)